sys-fs/cryfs/cryfs-0.9.7.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
```

Comment:
Have you tested every single one of them wrt https://wiki.gentoo.org/wiki/Project:Python/Adding_and_Removing_Python_implementations#Testing_support_for_the_implementation?